src/os/bluestore/BlueFS.cc

```diff
@@ -1993,6 +1995,9 @@ int BlueFS::_allocate(uint8_t id, uint64_t len,
         to_release.insert(p.offset, p.length);
       }
       alloc[id]->release(to_release);
+
+      if(logger && (id != BDEV_SLOW))
```

Comment:
I think we can avoid checking for non-null logger here.

Reply:
IMO this case has nothing with fragmentation - it's just lack of space at the corresponding volume hence suggest to rename the counter